src/renderer/components/+cluster-settings/components/install-user-mode.tsx

```diff
@@ -101,48 +85,51 @@ export class InstallUserMode extends React.Component<Props, State> {
     return buttons;
   }
 
-    @autobind()
+  @autobind()
   async installMetrics() {
     const { cluster } = this.props;
     console.log(`installing metrics onto ${cluster.preferences.clusterName}`);
 
     try {
-      this.setState({ status: ActionStatus.PROCESSING });
+      this.status = ActionStatus.PROCESSING
       const res = await clusterIpc.installFeature.invokeFromRenderer(cluster.id, UserModeFeature.id);
       console.log("installer response:", res);
-      this.setState({ status: ActionStatus.IDLE });
+      this.status = ActionStatus.IDLE
     } catch (err) {
-      this.setState({ status: ActionStatus.ERROR, errorText: err.toString() })
+      this.status = ActionStatus.ERROR
+      this.errorText = err.toString()
     }
   }
 
-    @autobind()
-    async uninstallMetrics() {
-      const { cluster } = this.props;
-      console.log(`uninstalling metrics from ${cluster.preferences.clusterName}`);
-
-      try {
-        this.setState({ status: ActionStatus.PROCESSING });
-        const res = await clusterIpc.uninstallFeature.invokeFromRenderer(cluster.id, UserModeFeature.id);
-        console.log("installer response:", res);
-        this.setState({ status: ActionStatus.IDLE });
-      } catch (err) {
-        this.setState({ status: ActionStatus.ERROR, errorText: err.toString() })
-      }
+  @autobind()
+  async uninstallMetrics() {
+    const { cluster } = this.props;
+    console.log(`uninstalling metrics from ${cluster.preferences.clusterName}`);
+
+    try {
+      this.status = ActionStatus.PROCESSING
+      const res = await clusterIpc.uninstallFeature.invokeFromRenderer(cluster.id, UserModeFeature.id);
+      console.log("installer response:", res);
```

Comment:
Console.log's are planning to stay here?

Reply:
No, but I am currently getting errors from these two buttons which I am trying to track down